calculator/src/Components/Buttons.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Buttons.css'; 
+
+const Buttons = ({handleInput, handleEqual, handleClear, handleOp, handleZero }) =>  {
+   
+   return (
+        <div className="calculator-container">
+          
+          <div className ="row">
+                <button  onClick={handleClear}className="buttons" value="clear">AC</button> 
+                <button className="buttons">+/-</button>
+                <button className="buttons">%</button>
+                <button onClick={handleOp}className="buttons" id="operator" value='/'>÷</button>
+             </div>
+
+             <div className ="row">
```

Comment:
Think about adding a Button component. Or holding all these buttons in an itterable array.